pkg/policyfilter/rthooks/rthooks.go

```diff
@@ -92,15 +92,25 @@ func createContainerHook(_ context.Context, arg *rthooks.CreateContainerArg) err
 		return err
 	}
 
+	var containerFound bool
+	var container *corev1.ContainerStatus
 	namespace := pod.ObjectMeta.Namespace
+	pod, container, containerFound = arg.Watcher.FindContainer(containerID)
+	if !containerFound {
+		log.WithError(err).Warnf("failed to find container information %s, aborting hook.", containerID)
```

Comment:
I think "aborting hook" might be misleading, how about "failed to find container information %s, continuing with hook".

Edit: this is fixed in a subsequent patch, so it's OK.